.travis.yml

```diff
@@ -15,13 +16,22 @@ gemfile:
   - gemfiles/rails32.gemfile
   - gemfiles/rails40.gemfile
   - gemfiles/rails41.gemfile
+  - gemfiles/rails42.gemfile
 language: ruby
 matrix:
   allow_failures:
     - rvm: ruby-head
+  include:
+    - rvm: 2.2.2
+      gemfile: gemfiles/rails50.gemfile
+    - rvm: 2.3.0
+      gemfile: gemfiles/rails50.gemfile
 rvm:
   - 1.9.3
   - "2.0"
   - "2.1"
+  - 2.2.2
```

Comment:
Rails 5 requires Ruby >= 2.2.2. When you declare Ruby 2.2 in `travis.yml`, Travis currently installs Ruby 2.2.0. I could have gone with 2.2.4 but I don’t actually think it matters much. If the tests pass on 2.2.2 but fail on 2.2.4, that’s a problem with Ruby, not this library.